dimod/core/sampler.py

```diff
@@ -23,6 +23,10 @@
 or :meth:`~.Sampler.sample_qubo`. The :class:`.Sampler` class provides the complementary
 methods as mixins and ensures consistent responses.
 
+Implemented sample methods must accept, and warn on, unknown keyword arguments
+`**kwargs`. This means that all sample methods must have the `**kwargs`
+parameter and call the :meth:`~.Sampler.remove_unknown_kwargs` method.
```

Comment:
The user technically does not have to call the method. Maybe 
```
...must have `**kwargs` parameter. The :meth:`~.Sampler.remove_unknown_kwargs` convenience method is provided for this purpose.
```
or something like that?

Reply:
That's true. I've updated with this change.